src/types.ts

```diff
@@ -179,33 +179,33 @@ type Args<
 > = [Action, Payload, TWSCallback<ResponsePayload, Event>];
 
 export type TConnectionInfoArgs = Args<"connection_info", null, IConnectionInfo>
-export type TCheckElementsArgs = Args<"check_elements", number[], number[]>;
-export type TDeleteElementsArgs = Args<"delete_elements", number[], unknown>;
+export type TGetElementsTypesArgs = Args<"check_elements", number[], number[]>;
+export type TEraseElementsArgs = Args<"delete_elements", number[], unknown>;
 export type TKeynodesElementsArgs = Args<
   "keynodes",
   TKeynodesPayload,
   number[]
 >;
-export type TTemplateSearchArgs = Args<
+export type TSearchByTemplateArgs = Args<
   "search_template",
```

Comment:
what about those strings?

Reply:
These strings are part of SC-JSON. They will be changed in other PR within sc-machine